website/snippets/_new-sl-setup.md

```diff
@@ -1,31 +1,89 @@
+
+<a href="https://www.getdbt.com/contact" style={{ color: 'white', backgroundColor: '#66c2c2', padding: '4px 8px', borderRadius: '4px', textDecoration: 'none', display: 'inline-block' }}>Book a free live demo</a> to discover the full potential of dbt Cloud Enterprise.
+:::
+
+### 3. View connection detail
+1. Go back to the **Project details** page for connection details to connect to downstream tools.
+2. Copy and share the environment ID, service token, host, as well as the service token name to the relevant teams for BI connection set up. If your tool uses the GraphQL API, save the GraphQL API host information instead of the JDBC URL. 
+
+    For info on how to connect to other integrations, refer to [Available integrations](/docs/cloud-integrations/avail-sl-integrations).
 
 <Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-configure-example.jpg" width="50%" title="After configuring, you'll be provided with the connection details to connect to you downstream tools." />
 
-7. Save and copy your environment ID, service token, and host, which you'll need to use in the downstream tools. For more info on how to integrate with partner integrations, refer to [Available integrations](/docs/cloud-integrations/avail-sl-integrations).
+### 4. Add more credentials <Lifecycle status="enterprise"/>
+dbt Cloud Enterprise plans can optionally add multiple credentials and map them to service tokens, offering more granular control and tailored access for different teams and projects. This can then be shared to relevant teams for BI connection setup. These credentials have physical access to the raw data, meaning they control the data accessed by the Semantic Layer.
+
+We recommend configuring credentials and service tokens to reflect your teams and their roles. For example, create tokens or credentials that align with your team's needs, such as providing finance-related schemas to the Finance team.
+
+Note that: 
+- Admins can link multiple service tokens to a single credential within a project, but each service token can only be linked to one credential per project.
+- When you send a request through the APIs, the service token of the linked credential will follow access policies of the data used to build your semantic objects.
+- [Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE')}` aren't supported the dbt Semantic Layer yet. You must use the actual credentials instead.
+
+To add multiple credentials and map them to service tokens:
+
+1. After configuring your environment, on the **Credentials & service tokens** page click the **Add Semantic Layer credential** button to configure the credential for your data platform. 
+2. On the **Create New Semantic Layer Credential** page, you can create multiple credentials and map them to a service token.
+3. In the **Add credentials** section, fill in the data platform's credential fields. We recommend using “read-only” credentials.
```

Comment:
@rpourzand this might be a weird question but users wouldn't input the credential name anywhere right? Just the token? Just thinking of case sensitivity if they need to use the casing name anywhere

Reply:
Yes! Sorry for not clarifying

Reply:
Per the comment above, we captured this in the previous callout/workflow.